zfsbootmenu/help-files/132/zfsbootmenu.7.ansi

```diff
@@ -117,7 +117,7 @@
 
     [1mzbm.hookroot=<hookspec>[0m
 
-      Tell ZFSBootMenu to attempt to read any early-setup, setup or teardown hooks from the path specified by [33mhookspec[0m in addition
+      Tell ZFSBootMenu to attempt to read any early-setup, setup, load_key or teardown hooks from the path specified by [33mhookspec[0m in addition
```

Comment:
This line is too long - don't edit this file by hand.

Reply:
Ok, got it